SPECS/harfbuzz/harfbuzz.spec

```diff
@@ -1,46 +1,45 @@
-Summary:	opentype text shaping engine
-Name:		harfbuzz
-Version:	1.9.0
-Release:    4%{?dist}
-License:	MIT
-URL:		http://harfbuzz.org
-Group:		System Environment/Libraries
+Summary:        opentype text shaping engine
+Name:           harfbuzz
+Version:        2.6.4
+Release:        1%{?dist}
+License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:	    https://www.freedesktop.org/software/harfbuzz/release/%{name}-%{version}.tar.bz2
-BuildRequires:	glib-devel
-BuildRequires:	freetype
-BuildRequires:	freetype-devel
-Requires:	    glib
+Group:          System Environment/Libraries
+URL:            http://harfbuzz.org
+Source0:        https://www.freedesktop.org/software/harfbuzz/release/%{name}-%{version}.tar.xz
+BuildRequires:  freetype
```

Comment:
Can we drop the base packages? It seems that `-devel` packages depend on the base ones already.